onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3615,20 +3614,21 @@ def aten_max_pool2d(
     else:
         strides = stride
 
-    return _aten_max_pool2d_onnx(self, kernel_shape, strides, pads, dilations, ceil_mode)
+    return _aten_max_pool_onnx(self, kernel_shape, strides, pads, dilations, ceil_mode, 3)
 
 
-@torch_op("aten::max_pool2d", private=True)
-def _aten_max_pool2d_onnx(
+@torch_op("aten::max_pool", private=True)
```

Comment:
@fatcat-z @justinchuby @xiaowuhu Although `aten::max_pool` seems to be a fine custom opset name here, it does't actually exist. Should we come up with a general solution/rule for shared ONNX Function?

Reply:
Or private=True should already indicated that to users?

Reply:
I agree that it'd be better for us to register private functions to an actual aten name unless there is a huge benefit to share code. In this case I suggest duplicating the `_aten_max_pool2d_onnx` logic to a separeate `_aten_max_pool3d_onnx` for clarity.

Reply:
I do see why we want to share logic in `_aten_max_pool_with_indices_onnx` though...

Reply:
So this name doesn't _really_ matter for private functions. We may be able to get away from this slight deviation for now.